README.md

```diff
@@ -13,3 +13,100 @@
+
+   For example imagine you had an algorithm that was $T(n)=10log(n)$ and another algorithm that was $G(n)=100000log(n)$.
+   When we look at the time complexity of $T,G$ we instantly know they are exactly the same but if we were actually running both algorithms on a system then G would be big optimization problem and would end up taking way longer than T. That is to say even though the complexity is the same it is more practical to use the one with the smaller constant in a real-life siutation.
+
+   Thus we see one reason asymptotic complexity can be misleading.
+
+-Reason 2: Time complexity is somewhat general
+
+Time-complexity is essentially a mathematical set contained by some given criteria. Which is to say maybe we only want functions less than or equal to(Big O), greater than or equal to(Big-$\Omega$). The time complexity only tells us that any given function $T(n)$ for some algorithm fits a criteria of a set, and because a lot of algorithms can fit into that set it doesn't tell us much about how practical it is, or how it works.
+
+For example:
+
+ Maybe your implementing something into a database and you need store information efficiently with respect to memory, so you choose an in-place algorithm to use less memory. However you also notice that its asymtotic complexity is worse than that of an alternative non-in-place algorithm so you decide to go with the algorithm with better time complexity instead. You use this algorithm and eventually you start to run out of memory because your not storing your data-operations efficiently. You then switch back to the slower algorithm and everything works out, becauase your now storing data efficiently. Which goes to show that a "good" time complexity doesn't always mean you get what you want.
+
+ This is a good example of why the general nature of asymtotic complexity doesn't reaveal much about its practically, especially because algorithms can and need to be very specific for different goals.
+
+Reason 3: A good time complexity doesn't imply a good outcome for every siutation
+
+Lastly to build upon our second reason, asymptotic complexity can be misleading when applied to certain situations such as sorting short lists.
+
+In this last example I want to pull upon the asymptotic complexity of insertion sort($O(n^2)$) and quick-sort($O(nlog(n))$). Generally Quick-Sort is considered one of the fastest, if not the fastest sorting algorithms. However this fastestness only applies to large lists and if you were working on smaller lists you would want to actually want to use insertion sort because its actually more effiective. This is mostly because insertion sort is more memory efficient and doesn't require any complex procedures. Whereas quicksort is a little more complex and while implace does take up more memory due to recursion. When comparing small or sorted lists like this we have that quicksort has $O(n^2)$ and insertion sort has $O(n)$.
```

Comment:
I'm not sure that this is true -- can you show that insertion sort is faster for small lists?